app/com/m3/octoparts/aggregator/service/PartRequestServiceBase.scala

```diff
@@ -33,8 +40,8 @@ trait PartRequestServiceBase extends RequestParamSupport {
    * @param pReq Part Request
    * @return Future[PartResponse]
    */
-  def responseFor(pReq: PartRequestInfo): Future[PartResponse] = {
-    val fMaybeCi = repository.findConfigByPartId(pReq.partRequest.partId)
+  def responseFor(pReq: PartRequestInfo)(implicit parentSpan: Span): Future[PartResponse] = {
+    val fMaybeCi = repository.findConfigByPartId(pReq.partRequest.partId).trace(s"find-config-by-id-${pReq.partRequest.partId}")
```

Comment:
most of the time this is going to hit the local cache (and the trace will be meaningless). Shall we add traces in the `DBConfigsRepository` instead?